engine/src/main/java/org/terasology/world/block/internal/BlockBuilder.java

```diff
@@ -193,7 +194,7 @@ private BlockAppearance createAppearance(BlockShape shape, Map<BlockPart, BlockT
             BlockPart targetPart = part.rotate(rot);
             textureAtlasPositions.put(targetPart, atlasPos);
             if (shape.getMeshPart(part) != null) {
-                meshParts.put(targetPart, shape.getMeshPart(part).rotate(rot.getQuat4f()).mapTexCoords(atlasPos, worldAtlas.getRelativeTileSize(), frameCount));
+                meshParts.put(targetPart, shape.getMeshPart(part).rotate(JomlUtil.from(rot.getQuat4f())).mapTexCoords(JomlUtil.from(atlasPos), worldAtlas.getRelativeTileSize(), frameCount));
```

Comment:
Some line breaks would be nice.
This applies to multiple changes made in this PR, do we have a convention about how many characters per line are acceptable?

Reply:
That's a little wide, yeah. The project standard (defined in Checkstyle config) is 175, which is pretty wide, so apply best judgement even if approaching that. This exceeds it by a bit.